### ISSUE TITLE
Multilingual plugin required

### ISSUE THREAD

**Issue by maiertech** [octopress/feeds#13]
The README makes me believe that `linkblog` and `multilingual` are optional plugins. Without the `multilingual` plugin I get the following exception:

```
  Liquid Exception: undefined method `lang' for #<Jekyll:Page @name="index.xml"> in _blog/archive.md
rake aborted!
NoMethodError: undefined method `lang' for #<Jekyll:Page @name="index.xml">
```

With `multilingual` added, no issue, everything works fine.

**Comment by imathis** [comment]
I've just released version 2.0 of this plugin. Now it supports category and tag RSS feeds, including multilingual support for all feed types. I've tested this and I believe this issue should be resolved now. Let me know if for some reason it isn't. Thanks!

**Comment by maiertech** [comment]
I can confirm this works fine now. Thanks!
